src/Symfony/Component/VarDumper/Server/Connection.php

```diff
@@ -9,104 +9,88 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\VarDumper\Dumper;
+namespace Symfony\Component\VarDumper\Server;
 
 use Symfony\Component\VarDumper\Cloner\Data;
-use Symfony\Component\VarDumper\Dumper\ContextProvider\ContextProviderInterface;
+use Symfony\Component\VarDumper\Server\ContextProvider\ContextProviderInterface;
 
 /**
- * ServerDumper forwards serialized Data clones to a server.
+ * Forwards serialized Data clones to a server.
  *
  * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
  */
-class ServerDumper implements DataDumperInterface
+class Connection
```

Comment:
I suggest marking this class as `@internal`

Reply:
With the suggested code, marking it `@internal` would mean someone using the component out of symfony full-stack cannot reliably wire this in his own application.